### ISSUE TITLE
Converting unstructured ingress

### ISSUE THREAD

**Issue by LenglBoy** [vmware-tanzu/octant#2283]
While navigating to a pods "Recourse Viewer" the following errormessage appeared:

> create resource viewer: unable to visit /v1, Kind=Pod PODNAME: error unable to visit object </v1, Kind=Pod PODNAME>: pod </v1, Kind=Pod PODNAME> visit service </v1, Kind=Service auth-service>: converting unstructured ingress: scheme convert: converting (v1beta1.Ingress) to (v1.Ingress): unknown conversion

The console printed the following related message:
```
ERROR   api/content_manager.go:158      generate content        {"client-id": "CLIENTID", "err": "generate content: scheme convert: converting (v1beta1.Ingress) to (v1.Ingress): unknown conversion", "content-path": "overview/namespace/auth"}
github.com/vmware-tanzu/octant/internal/api.(*ContentManager).runUpdate.func1
        /__w/octant/octant/internal/api/content_manager.go:158
github.com/vmware-tanzu/octant/internal/api.(*InterruptiblePoller).Run.func1
        /__w/octant/octant/internal/api/poller.go:86
github.com/vmware-tanzu/octant/internal/api.(*InterruptiblePoller).Run
        /__w/octant/octant/internal/api/poller.go:95
github.com/vmware-tanzu/octant/internal/api.(*ContentManager).Start
        /__w/octant/octant/internal/api/content_manager.go:132
```

**Comment by ftovaro** [comment]
I'm gonna work on this one

**Comment by wwitzel3** [comment]
Given our version skew support, https://github.com/vmware-tanzu/octant#supported-version-skew we will want to directly support v1beta1.Ingress until the current release of Kubernetes is 0.24.

See for when it will be removed - https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

**Comment by ftovaro** [comment]
Still working on this issue. 
There is another related issue [here](https://github.com/vmware-tanzu/octant/issues/2063).

With @xtreme-vikram-yadav  we are working on a solution that solves both issues.

**Comment by wwitzel3** [comment]
I believe @xtreme-vikram-yadav has been working on this and might even have a branch. There are some details about the fix/solution in #2063. I'm closing this as a duplicate of #2063
